### PR TITLE
fix: add discord icon

### PR DESCRIPTION
**Pull request by MuhammadArslanKhan96**
[Description]:

Added discord icon in social links at navigation bar and footer

### CODE REVIEW

**Comment by zolotokrylin** [comment]
@MuhammadArslanKhan96 can you isolate please the work related to discord from the rest of the changes in this PR. 
I could quickly merge the icon-related work, but your refactoring needs more review and it is mixed together with Discord which should not be. I can see code related to table render being involved, etc. It is great that you have initiative, but let's solve one problem at a time. Feel free to create another problem related to your solution in our repository. 
Basically: 
1. create another PR with discord-only related changes
2. creat an issue in the form of a Problem statement in our repository and attach this PR.

I hope I am clear. Otherwise, ask questions if any. Thanks

**Review by zolotokrylin — CHANGES_REQUESTED** [review]
@MuhammadArslanKhan96 can you isolate please the work related to discord from the rest of the changes in this PR. 
I could quickly merge the icon-related work, but your refactoring needs more review and it is mixed together with Discord which should not be. I can see code related to table render being involved, etc. It is great that you have initiative, but let's solve one problem at a time. Feel free to create another problem related to your solution in our repository. 
Basically: 
1. create another PR with discord-only related changes
2. creat an issue in the form of a Problem statement in our repository and attach this PR.

I hope I am clear. Otherwise, ask questions if any. Thanks

**Comment by MuhammadArslanKhan96** [comment]
@zolotokrylin problem is in base branch, because other members also created PR for stage base branch, that's why other members code including here, please have a look now, I've changed base branch stage to main, it works for you?

I've tried to create another PR but when I'm selecting base branch stage then I'm getting previous PR again

**Review by zolotokrylin — APPROVED** [review]
Yes, it works!

**Comment by predrag-us** [comment]
> @MuhammadArslanKhan96 is attempting to deploy a commit to the **Holdex Accelerator** Team on [Vercel](https://vercel.com).
> 
> A member of the Team first needs to [authorize it](https://vercel.com/git/authorize?team=Holdex%20Accelerator&type=github&job=%7B%22headInfo%22%3A%7B%22sha%22%3A%22d2534dcc9f10265d5935bbbbfe849e9371cb9499%22%7D%2C%22id%22%3A%22QmTRk5qQk6wyzqzfJnYKguceKbhFfowLogUdZnB4Z7kWhx%22%2C%22org%22%3A%22holdex%22%2C%22prId%22%3A455%2C%22repo%22%3A%22holdex-venture-studio%22%7D).

@zolotokrylin @MuhammadArslanKhan96 

I can't review it since there's no Vercel Preview available.

**Comment by MuhammadArslanKhan96** [comment]
@predrag-us it's merged in main branch by @zolotokrylin now you can preview this directly on holdex.io

**Comment by predrag-us** [comment]
@MuhammadArslanKhan96 Please check the icon sizing in the footer.

**Comment by markholdex** [comment]
@predrag-us please report as problem statement because current PR and issue is closed. Thanks.

**Comment by MuhammadArslanKhan96** [comment]
@predrag-us fixed, please check

https://github.com/holdex/holdex-venture-studio/pull/457

**Comment by predrag-us** [comment]
Problem: The `Discord` icon sizing in the `Footer` is incorrect
